workshops/3D Heatmap/README.md

```diff
@@ -0,0 +1,202 @@
+---
```

Comment:
Good job I left some various comments. Its not ready to be a workshop yet, but if change a few things it probably could be a workshop.

Reply:
Thanks, for helping me out with all changes done.